save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -35,21 +38,23 @@ class CloneRepositoryController(
      * @return mono string
      */
     @PostMapping(value = ["/submitExecutionRequest"])
-    fun submitExecutionRequest(@RequestBody executionRequest: ExecutionRequest): Mono<ResponseEntity<*>>? {
+    fun submitExecutionRequest(@RequestBody executionRequest: ExecutionRequest): Response {
         val project = executionRequest.project
+        var projectId: String? = null
         try {
-            projectService.saveProject(project)
-            log.info("Project ${project.id} saved")
+            projectId = projectService.saveProject(project)?.toString() ?: "With non Id"
```

Comment:
If project is saved, then an id is assigned to it, so `saveProject` should never return null